docs/v0.9/en/getting-started/installation/greptimedb-cluster.md

```diff
@@ -1,73 +1,150 @@
 # GreptimeDB Cluster
 
-The GreptimeDB cluster can run in the [cluster](/contributor-guide/overview.md) mode to scale horizontally.
+The GreptimeDB cluster can run in cluster mode to scale horizontally.
 
-## Install gtctl and run playground
+## Use Docker Compose
 
-[gtctl](https://github.com/GreptimeTeam/gtctl) is the cli tool for managing the GreptimeDB cluster. You can use the following one-line installation(only for Linux and macOS):
+###  Prerequisites
+
+Using Docker Compose is the easiest way to run the GreptimeDB cluster. Before starting, make sure you have already installed the Docker.
+
+### Step1: Download the YAML file for Docker Compose
 
 ```
-curl -fsSL \
-  https://raw.githubusercontent.com/greptimeteam/gtctl/develop/hack/install.sh |\
-  sh
+wget https://raw.githubusercontent.com/GreptimeTeam/greptimedb/<%greptimedb-version%>/docker/docker-compose/cluster-with-etcd.yaml
 ```
 
-:::tip Note
+### Step2: Start the cluster
 
-For Windows users, due to the complexity and compatibility of running multiple components together, we strongly recommend enabling WSL([Windows Subsystem for Linux](https://learn.microsoft.com/en-us/windows/wsl/about)) on your operating system, and lunch a latest Ubuntu to proceed.
+```
+GREPTIMEDB_VERSION=<%greptimedb-version%> \
+  docker compose -f ./cluster-with-etcd.yaml up 
+```
 
-:::
+If the cluster starts successfully, it will listen on 4000-4003. , you can access the cluster by referencing the [Quick Start](../quick-start.md).
 
-Once the download is completed, the binary file `gtctl` will be stored in your current directory.
+### Clean up
 
-The **fastest** way to experience the GreptimeDB cluster is to use the playground:
+You can use the following command to stop the cluster:
 
 ```
-./gtctl playground
+docker compose -f ./cluster-with-etcd.yaml down
 ```
 
-When the command is executed, the playground will be started in the foreground. You can use `Ctrl+C` to stop the playground. The playground will deploy the minimal GreptimeDB cluster in bare-metal mode on your host.
+By default, the data will be stored in `/tmp/greptimedb-cluster-docker-compose`. You also can remove the data directory if you want to clean up the data:
 
-If everything is ok, you can visit the Dashboard via `http://localhost:4000/dashboard/`.
-
-For more details, please refer to [gtctl operations](/reference/gtctl.md).
+```
+rm -rf /tmp/greptimedb-cluster-docker-compose
+```
 
 ## Deploy the GreptimeDB cluster in Kubernetes
 
-If your Kubernetes cluster is ready(a 1.18 or higher version is required), you can use the following command to deploy the GreptimeDB cluster:
+We are using [GreptimeTeam/helm-charts](https://github.com/GreptimeTeam/helm-charts) to deploy the GreptimeDB cluster in Kubernetes. For production usage, we highly recommend deploying it in Kubernetes.
```

Comment:
please do not write the same content twice.
It is recommended to delete all the content below, and add a link to [the k8s deployment in the operations section](https://docs.greptime.com/user-guide/operations/deploy-on-kubernetes/overview).

Reply:
oops...Let me change it.